src/main/kotlin/com/exactpro/th2/common/schema/message/impl/rabbitmq/transport/Message.kt

```diff
@@ -25,7 +25,7 @@ interface Message<T> : Cleanable {
     /** The metadata is not mutable by default */
     var metadata: MutableMap<String, String>
     var protocol: String
-    var body: T
+    val body: T
```

Comment:
Why we close set option? It looks strange on the other mutable fields background

Reply:
You're right, probably, we should return the mutability here...
At least until we migrate to immutable objects (if we do)